testdata/expected/program/python-single-module-lib/single-module-lib/PipPackage.graph.json

```diff
@@ -1,104 +0,0 @@
-{
```

Comment:
where did this file go? It's a program (i.e., non-docker) testcase, so shouldn't it have reappeared in `testdata/expected/python-single-module-lib` directory?